packages/drivers/routerlicious-driver/src/test/restWrapper.spec.ts

```diff
@@ -7,67 +7,49 @@ import assert from "assert";
 import { TelemetryUTLogger } from "@fluidframework/telemetry-utils";
 import { DriverErrorType } from "@fluidframework/driver-definitions";
 import { RateLimiter } from "@fluidframework/driver-utils";
-import fetchMock from "fetch-mock";
-import * as nodeFetchModule from "node-fetch";
-import { reset, replace } from "sinon";
 import { RouterliciousRestWrapper } from "../restWrapper";
 import { R11sErrorType } from "../errorUtils";
+// eslint-disable-next-line @typescript-eslint/no-require-imports,import/order
```

Comment:
nit: do you have to disable import/order?

Reply:
Ope, yeah my VSCode wasn't recognizing that esModuleInterop: true was set. Switched this to a normal import and put it in order.